src/test/java/com/meta/chatbridge/configuration/RootConfigurationTest.java

```diff
@@ -103,6 +134,36 @@ void valid(@TempDir Path dir) throws IOException {
     config.toServicesRunner();
   }
 
+  @Test
+  void validWA(@TempDir Path dir) throws IOException {
+    Path configFile = dir.resolve("config.toml");
+    Files.writeString(configFile, TOML_WA);
```

Comment:
Do we want test config files as part of the repo at some point? Or is it better practice to always write to a new file like this - and in the case someone adds a toml config file to the directory, does that break this test?

Reply:
so I absolutely need more tests around this, however that's a good point. In general it's always a better practice to write tests that encompass as many elements as possible (there are exceptions to this) but yes that would be a good practice to write it all the way to disk and ensure that a service can/cannot be formed